server/service.go

```diff
@@ -287,7 +287,7 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 	}
 
 	var mu sync.Mutex
-	relays := make(map[string][]string) // relays per blockHash
+	relays := make(map[string][]RelayEntry) // relays per blockHash
```

Comment:
This makes me happy. Should the comment go?
I could sign off to `relays := make(map[BlockHash][]RelayEntry)`

Reply:
done